pkg/rc/replication_controller.go

```diff
@@ -55,6 +56,12 @@ type kpStore interface {
 		nodeName string,
 		manifestID types.PodID,
 	) (time.Duration, error)
+	Pod(podPrefix kp.PodPrefix,
+		nodename string,
```

Comment:
The actual `kp.Store` interface uses `nodename`—I'm not sure why the other methods here want to be different.